android/src/com/mapswithme/maps/gallery/Image.java

```diff
@@ -0,0 +1,128 @@
+package com.mapswithme.maps.gallery;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Image implements Parcelable
+{
+  private final String url;
```

Comment:
Please, be self-consistent - use the same naming rules as before. I.e. mUrl, mSmallUrl, mDescription, etc.